src/webprojects/README.md

```diff
@@ -6,4 +6,5 @@ This set of documentation contains basic information on localizing and testing w
 * [Addons.mozilla.org (AMO)](amo.md).
 * [Developer.mozilla.org (MDN)](mdn.md).
 * [Mozilla Donate website](fundraising.md).
+* [Firefox Relay Website & Firefox Relay add-on](webprojects/relay.md)
```

Comment:
Please shorten this, e.g. `Firefox Relay website & add-on`. Also, capitalization consistency.